src/datamodule.py

```diff
@@ -106,6 +108,83 @@ def __len__(self):
         return len(self.chips_path)
 
 
+class ClayDatasetV1(Dataset, ClayDataset):
+    def __init__(self, mountpath: str, index_path: Path, transform=None):
+        super().__init__()
+        self.mountpath = mountpath
+        self.index = gads.dataset(index_path, format="parquet")
+        self.table = self.index.to_table()
+        self.transform = transform
+
+    def read_chip(self, row: int):
+        chip_index_x = self.table.column("chip_index_x")[row].as_py()
+        chip_index_y = self.table.column("chip_index_y")[row].as_py()
+
+        chipper = Chipper(
+            mountpath=self.mountpath,
+            platform=self.table.column("platform")[row],
+            item_id=self.table.column("item")[row],
```

Comment:
I'm getting a KeyError when reading the parquet tables from `s3://clay-v1-data/index/sentinel-2-l2a/**/index_*.parquet`:

```
KeyError: 'Field "platform" does not exist in schema'
KeyError: 'Field "item" does not exist in schema'
```
This is what the `pyarrow.Table` looks like:
```
pyarrow.Table
chipid: string
date: date32[day]
chip_index_x: uint16
chip_index_y: uint16
cloud_cover_percentage: float
nodata_percentage: float
geometry: binary
----
chipid: [["S2A_10SEJ_20190331_0_L2A-1-0","S2A_10SEJ_20190331_0_L2A-2-0","S2A_10SEJ_20190331_0_L2A-3-0","S2A_10SEJ_20190331_0_L2A-4-0","S2A_10SEJ_20190331_0_L2A-5-0",...,"S2A_10SEJ_20190331_0_L2A-37-41","S2A_10SEJ_20190331_0_L2A-38-41","S2A_10SEJ_20190331_0_L2A-39-41","S2A_10SEJ_20190331_0_L2A-40-41","S2A_10SEJ_20190331_0_L2A-41-41"]]
date: [[2019-03-31,2019-03-31,2019-03-31,2019-03-31,2019-03-31,...,2019-03-31,2019-03-31,2019-03-31,2019-03-31,2019-03-31]]
chip_index_x: [[1,2,3,4,5,...,37,38,39,40,41]]
chip_index_y: [[0,0,0,0,0,...,41,41,41,41,41]]
cloud_cover_percentage: [[0,0,0.016845703,0.016052246,0.06036377,...,0.00012207031,0.000061035156,0.00024414062,0.000061035156,0.000061035156]]
nodata_percentage: [[0.45947266,0,0,0,0,...,0,0,0,0,0]]
geometry: [[01030000000100000005000000A85F83A530BC5EC0E5337B4708E04340C26DBCF830BC5EC0D864CC7214DD4340A733C5651ABE5EC0406802D314DD4340E970523C1ABE5EC07D43C5A708E04340A85F83A530BC5EC0E5337B4708E04340,01030000000100000005000000E0D7CC0E47BA5EC0A1C2A9A607E043405229CC8B47BA5EC01C6F1CD213DD4340C26DBCF830BC5EC0D864CC7214DD4340A85F83A530BC5EC0E5337B4708E04340E0D7CC0E47BA5EC0A1C2A9A607E04340,01030000000100000005000000972A3B785DB85EC0A5FB50C506E0434054B3001F5EB85EC0FF92F2F012DD43405229CC8B47BA5EC01C6F1CD213DD4340E0D7CC0E47BA5EC0A1C2A9A607E04340972A3B785DB85EC0A5FB50C506E04340,01030000000100000005000000D5A8DAE173B65EC0B1EF70A305E04340C75866B274B65EC03BE14ECF11DD434054B3001F5EB85EC0FF92F2F012DD4340972A3B785DB85EC0A5FB50C506E04340D5A8DAE173B65EC0B1EF70A305E04340,0103000000010000000500000090A3B74B8AB45EC051B4094104E043409B6609468BB45EC0586F316D10DD4340C75866B274B65EC03BE14ECF11DD4340D5A8DAE173B65EC0B1EF70A305E0434090A3B74B8AB45EC051B4094104E04340,...,010300000001000000050000008B89CD7E51785EC0CE7C5154496643407766696857785EC08CB7E3845563434089DA5B2D3A7A5EC0C9D110A65E634340A84C906B347A5EC075EA6877526643408B89CD7E51785EC0CE7C515449664340,010300000001000000050000000FDAC6936E765EC01832DFF23F664340747832A574765EC06FD168254C6343407766696857785EC08CB7E384556343408B89CD7E51785EC0CE7C5154496643400FDAC6936E765EC01832DFF23F664340,0103000000010000000500000002EB87AA8B745EC063B912533666434098B9C2E391745EC03ECEA08742634340747832A574765EC06FD168254C6343400FDAC6936E765EC01832DFF23F66434002EB87AA8B745EC063B9125336664340,01030000000100000005000000EF681CC3A8725EC03DC6EC742C664340B5D22524AF725EC058618CAB3863434098B9C2E391745EC03ECEA0874263434002EB87AA8B745EC063B9125336664340EF681CC3A8725EC03DC6EC742C664340,010300000001000000050000001B0090DDC5705EC0CB106E58226643405A6C6766CC705EC09D422C912E634340B5D22524AF725EC058618CAB38634340EF681CC3A8725EC03DC6EC742C6643401B0090DDC5705EC0CB106E5822664340]]
```
Are these `platform` and `item` keys meant to be in the parquet table? Or maybe I'm looking at the wrong index parquet file?